samples/src/Microsoft.Azure.IIoT.App/src/Startup.cs

```diff
@@ -150,7 +151,7 @@ public class Startup {
             });
 
             services.AddAntiforgery(options => {
-                options.Cookie.SameSite = SameSiteMode.None;
+                options.Cookie.SameSite = SameSiteMode.Strict;
```

Comment:
Please build and deploy from your branch - run "acrbld <name of deployment>" then try using the app from there.   This is a source of failures in the engineering tool when running in container (i.e. deploy.ps1 script).

Reply:
revert to none. I will do more test